deltas/tokenizers/wikitext_split.py

```diff
@@ -1,4 +1,5 @@
 from .tokenizer import RegexTokenizer
+from .lexikon_chars import *
```

Comment:
This should be explicit.  I prefer something like `from . import lexicon_chars` and then to reference the variables like `lexicon_chars.cjk` in the code.

Reply:
done